charts/argo-cd/templates/crds/crd-application.yaml

```
@@ -1,3 +1,4 @@
+{{- if .Values.crds.install }}
 apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
```

Let's also implement `crds.keep` option as argo-events [does](https://github.com/argoproj/argo-helm/blob/argo-events-2.0.3/charts/argo-events/templates/crds/eventbus-crd.yml#L7-L9).

```suggestion
metadata:
  annotations:
    {{- if .Values.crds.keep }}
    "helm.sh/resource-policy": keep
    {{- end }}
```

---

Implemented it for you @yu-croco

---

Thank you!
I will check failed CI. 👀